bears/general/AnnotationBear.py

```diff
@@ -23,10 +25,12 @@ def run(self, filename, file, language: str, coalang_dir: str = None):
         :param coalang_dir:
             External directory for coalang file.
         :return:
-            One HiddenResult containing a dictionary with keys being 'strings'
-            or 'comments' and values being a tuple of SourceRanges pointing to
-            the strings and a tuple of SourceRanges pointing to all comments
-            respectively. The ranges do include string quotes or the comment
+            One HiddenResult containing a dictionary with keys being
+            'singleline strings', 'multiline strings', 'singleline comments'
+            and 'multiline comments' and values being a dictionary with
+            keys ```'start_delimiter_range', 'end_delimiter_range'
```

Comment:
can you use single backslaches around every key? Also for the keys above please, be consistent

Reply:
@aptrishu actually double backticks would be correct here